Specs/Scene/ImplicitSubtreeSpec.js

```diff
@@ -2932,5 +2932,118 @@ describe("Scene/ImplicitSubtree", function () {
         }
       });
     });
+
+    it("handles legacy 3DTILES_metadata schema correctly for arrays and strings", function () {
```

Comment:
Maybe add this test for the new schema as well? (so same test but `useLegacySchema: false`)

Reply:
See the test above this one, it should be what you said. Though I changed the implicit tester to ensure that the default is `useLegacySchema=false`

Reply:
oops! missed that, thanks for the correction